### DISCUSSION TITLE
glTF extension KHR_techniques_webgl is not supported

### DISCUSSION THREAD

**Discussion by hcshen** [atteneder/glTFast#542] [General]
I encountered this error while loading 3dtiles data. Can I just add this type to the enumerated type?

**Comment by atteneder** [comment]
Hi @hcshen ,

The status of the KHR_techniques_webgl extension itself is it's an [archived draft](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Archived/KHR_techniques_webgl/README.md#status), thus supporting it is not planned.

What benefit would it bring to add it to `GLTFast.Extension` or `GLTFast.ExtensionName`?

**Reply by hcshen** [reply]
When I add this type directly to the enumeration, the model loads up but it's gray, no material.

**Reply by atteneder** [reply]
glTF files have two lists:

- used extensions
- required extensions

If KHR_techniques_webgl is in the list of required extensions, then refusing to load the file is expected behavior. If it only is in the list of used extensions, then I'd consider this a bug.

Can you provide a test file to reproduce this behavior so I can look into it?

Which enum exactly do you add KHR_techniques_webgl to?
